src/css/controls/flags/ads.less

```diff
@@ -53,9 +53,13 @@
             }
         }
 
+        .jw-fullscreen-ads {
+            display: flex;
+        }
+
         .jw-tooltip,
         .jw-icon-tooltip:not(.jw-icon-volume),
-        .jw-icon-inline:not(.jw-icon-playback):not(.jw-icon-fullscreen):not(.jw-icon-volume) {
+        .jw-icon-inline:not(.jw-icon-playback):not(.jw-fullscreen-ads):not(.jw-icon-volume) {
```

Comment:
Is it okay that we will be changing the fullscreen placement for all ad types? Googima had to move to not clash with the IMA controls, but we usually have it in the lower right corner if we can. I'm cool with the change, just wondering.

Reply:
In all types we were using flex order to re-order it the same way. Now it's actually placed there to begin with

Reply:
Just trying to understand, are we now applying a new class `jw-fullscreen-ads` to IMA only skip button, and we still use the original `jw-icon-fullscreen` class for both the new and the old fullscreen? Change here seems to display `jw-fullscreen-ads` icon without displaying `jw-icon-fullscreen` as default - is that correct?

Reply:
I thought the flex order was just being used in IMA (which the related PR in IMA is removing). All the other ad types default to our control layout which has the icon in the right corner. Not against the change, just wanted to mention that.

Reply:
Ah you're right. It's just IMA

Reply:
These kinds of insights are exactly why I posted this as a draft. I'll circle back and see if I can gate this changes to only IMA